src/getHumanAge.test.js

```diff
@@ -1,8 +1,61 @@
+'use strict';
+
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
   test('should be declared', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return [0, 0] for 0 cat's and dog's years`, () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test(`should return 0 of human age for cat's and dog's years less than 15`,
+    () => {
+      expect(getHumanAge(14, 14))
+        .toEqual([0, 0]);
+    });
+
+  test(`should return 1 of human year for 15 cat's and dog's years`, () => {
```

Comment:
```suggestion
  test(`should return [1, 1] of human year for 15 cat's and dog's years`, () => {
```

fix it everywhere :)